### DISCUSSION TITLE
Is it possible to add colour to the groupname_col of a tab_stub too?

### DISCUSSION THREAD

**Discussion by HansMellman** [posit-dev/great-tables#443] [Q&A]
Does anyone know if it's possible to add colour to the groupname_col of a tab_stub too? I've tried utilising the tab_style on the specified column both before and after using it in the tab_stub but it doesn't apply the colouring - would love to be able to colour those.

Thanks!

![uFbKWvyagN](https://github.com/user-attachments/assets/305b0ae8-d2f0-47f2-bdd6-621a3b43b4cd)

**Comment by jrycw** [comment]
@HansMellman, maybe try using [GT.tab_options](https://posit-dev.github.io/great-tables/reference/GT.tab_options.html#great_tables.GT.tab_options) like:
```python
from great_tables import GT
from great_tables.data import gtcars


gtcars_mini = gtcars.head(5).loc[:, ["mfr", "model", "year"]]
(
    GT(gtcars_mini)
    .tab_stub(groupname_col="mfr", rowname_col="model")
    .tab_options(row_group_background_color="green")
)
```
![image](https://github.com/user-attachments/assets/8c9910fb-dbea-41c5-b98c-c43e9e6ba951)

**Comment by HansMellman** [comment]
Ah awesome, thanks so much - do you know if it's possible to specify an index for the colour? as in, to perhaps have the first 3 rows of the group one colour and the second 3 rows a different colour?

**Reply by jrycw** [reply]
I don’t have an answer at the moment, but I believe we may need to wait until the implementation of all the `Loc` objects (#341) is completed to fully meet your intention.

**Reply by HansMellman** [reply]
Thanks so much for your help on this one - I appreciate it. Looking forward to seeing the evolution of this package.
